contributing/community/speaker-mentoring.rst

```diff
@@ -0,0 +1,43 @@
+Speaker mentoring
+=================
+
+The Symfony community benefits greatly from as many people as possible
+sharing their knowledge and experience with others. Every different
+point of view adds to our collective understanding of how to best use
+and evolve the code, design patterns and architecture provided within
+the Symfony community. As such we specifically welcome not only sharing
+from long time contributors, but also from new users as they will come
+across entirely different challenges with a totally fresh new look and
+perspective.
+
+How to get started
+------------------
+
+Giving a first talk at a conference can seem quite intimidating but we
+want to make sure that as many people as possible are empowered to take
+this path if they are motivated. We have collected a few resources with
+advice to get started. More importantly we connect experienced speakers
+with people just taking their first steps in this area:
+
+.. tip::
+
+    A good first step might be giving a talk at a local user group to a
+    smaller crowd that one knows more intimately. As a next step give
+    a talk at conference in your mother tongue.
+
+The best way to find people that can review your talk idea or slides is
+the `speaker mentoring channel`_ on Symfony Devs slack. There are many
+seasoned speakers with knowledge in various parts of Symfony that are
+motivated to help anyone get started on the path towards becoming a
+public speaker. They can even do practice runs via video chat!
+Furthermore, they can also be an ally when it comes to the day of
+giving the talk at a conference!
+
+A great resource with advice on everything related to`public speaking`_
```

Comment:
As an FYI, this sentence is of a passive voice, maybe it could be rewritten into an active voice. But I myself am no star in grammar and copyrighting.

Reply:
To me, the passive voice gives this part a bit of a footnote feeling. It sounds good as is: not something you _should_ do, but some more information in case you're interested.